controllers/userController.js

```diff
@@ -1,35 +1,28 @@
 const userService = require('../services/userService');
+const { catchAsync } = require('../utils/error');
 
-const signUp = async (req, res) => {
-  try {
-    const { email, password, firstname, lastname, skintype } = req.body;
+const signUp = catchAsync(async (req, res) => {
+  const { email, password, firstname, lastname, skintype } = req.body;
 
-    if (!email || !password || !firstname || !lastname) {
-      throw new Error('KEY Error!');
-    }
-
-    await userService.signUp(email, password, firstname, lastname, skintype);
-
-    return res.status(201).send({ message: 'SIGNUP_SUCCESS' });
-  } catch (err) {
-    return res.status(err.statusCode || 400).json({ message: err.message });
+  if (!email || !password || !firstname || !lastname) {
+    throw new Error('KEY Error!');
   }
-};
 
-const signIn = async (req, res) => {
-  try {
-    const { email, password } = req.body;
+  await userService.signUp(email, password, firstname, lastname, skintype);
 
-    if (!email || !password) {
-      throw new Error('KEY Error!');
-    }
-    await userService.signIn(email, password);
+  return res.status(201).send({ message: 'SIGNUP_SUCCESS' });
+});
 
-    return res.status(200).json('SIGNIN SUCCESS!');
-  } catch (err) {
-    return res.status(err.statusCode || 400).json({ message: err.message });
+const signIn = catchAsync(async (req, res) => {
+  const { email, password } = req.body;
+
+  if (!email || !password) {
+    throw new Error('KEY Error!');
   }
-};
+  const result = await userService.signIn(email, password);
```

Comment:
나중에 리팩토링 할 때 변수명 수정해주세요
```suggestion
  const token = await userService.signIn(email, password);
```